sdcm/sct_config.py

```diff
@@ -204,13 +227,14 @@ class SCTConfiguration(dict):
 
     config_options = [
         dict(name="config_files", env="SCT_CONFIG_FILES", type=str_or_list_or_eval,
-             help="a list of config files that would be used"),
+             help="a list of config files that would be used", appendable=False),
```

Comment:
Do I understand it correctly that the `appendable` parameter is used only for docs?
Not limiting params possibilities?

Reply:
as @soyacz suggested, I'm adding a check to see some are not wrongly used